components/meta-packages/entire-incorporation/entire.p5m

```diff
@@ -10,12 +10,13 @@
 #
 
 #
-# Copyright 2014 Andrzej Szeszo.  All rights reserved.
+# Copyright 2021 Klaus Ziegler.  All rights reserved.
```

Comment:
This is clearly a formal issue. You can't revoke Andzej's rights if you don't rewrite this file completely.
Be careful about this.
@AndWac  @Toasterson

Reply:
From what I remember we cannot identify from a git diff if it's a new file with these lines changed or a changed line in the same file. I assumed it's a new file.